src/features/Description/Projects/getProjects.js

```diff
@@ -0,0 +1,11 @@
+export const getProjects = async () => {
+
+    const response = await fetch("https://api.themoviedb.org/3/person/3/movie_credits?api_key=9515ffc857c67f1558538dad140abb29&language=en-US")
```

Comment:
Everything is good and you get data but you have set idPeople=3 and it always be Harrison Ford. I think you could use useParam(). It helped me in movieDetails

Reply:
I confirm information from Kuba. Details for only one character are displayed.